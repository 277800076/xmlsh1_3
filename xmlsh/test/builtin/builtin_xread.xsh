# builtin xread command
# reads input into a single xml variable
# input is expected in XML format (not xquery format, no sequences)

xread a <<EOF
<test>data</test>
EOF

echo a is $a
echo type of a is 
set | xquery '//variable[@name="a"]/@type/string()'


