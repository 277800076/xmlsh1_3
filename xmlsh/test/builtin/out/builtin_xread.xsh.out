a is <test>data</test>
type of a is
xml